src/Mail/SendmailMailer.php

```diff
@@ -29,6 +29,9 @@ class SendmailMailer implements IMailer
 	 */
 	public function send(Message $mail): void
 	{
+		if (function_exists('mail') === false) {
+			throw new SendException('Unable to send email: mail() has been disabled for security reasons.');
```

Comment:
Personally, I would not print the reason. `has been disabled.` is IMHO enough.

Reply:
@milo fixed.

Reply:
https://github.com/mzk/mail/commits/disable-function-mail tady ten commit je, ale v pull requestu se neobjevil